server/src/main/resources/application.yml

```diff
@@ -15,6 +15,9 @@ spring:
     password: password
     initialization-mode: always
     platform: postgres
+  jackson:
+    serialization:
+      FAIL_ON_EMPTY_BEANS: false
```

Comment:
Brauchen wir wirklich die application.yml config *und* die config in der PersistanceConfig.java?